### PR TITLE
add info about using nvm to package-manaager.md

### PR DESCRIPTION
**Pull request by danbev**
Adding information about a way of using nvm to use version of
Node.js built from source.

### CODE REVIEW

File: locale/en/download/package-manager.md
Comment (ljharb) [inline locale/en/download/package-manager.md]: please use `nvm_version_path "$VERSION"` so as to not hardcode the path - i'm much more likely to break the path than I am to break that internal `nvm` function.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: locale/en/download/package-manager.md
Comment (danbev) [reply locale/en/download/package-manager.md]: @ljharb Sorry about this, I was not ignoring you suggestion but failed to get it to work for me. I'm not exactly sure about the cause for this not working yet but from within the Makefile the `nvm_version_path` function cannot be found. I'm not sure if this is because it is a function. I'll take another look tomorrow, but let me know if you have any ideas or suggestions on this. Thanks!

Comment (ljharb) [reply locale/en/download/package-manager.md]: It will only be found if `nvm.sh` is sourced, and `NVM_DIR` should only be set if the profile file is sourced (which sources `nvm.sh` for you). If necessary, you'll want to source `nvm.sh` in the Makefile

Comment (danbev) [reply locale/en/download/package-manager.md]: I don't think that we would want to source `nvm.sh` from the Makefile. I think the Makefile should stay unchanged and not have anything nvm specific. 

I've looked into exporting the function but this looks like it would not be supported by all shells. 

I'm suggesting that we change this to be `DESTDIR=~/.nvm/versions/node/` and then document that if nvm has been installed into a different location then the path must be updated to reflect that.

What do you think about that?

Comment (ljharb) [reply locale/en/download/package-manager.md]: The function itself is POSIX-compliant, so it should work in any POSIX-compliant shell.

I think that it depends on if your goal is correctness/robustness or convenience. Correctness requires using the same `NVM_DIR` env var that the user will use, and robustness requires not hardcoding the nvm dir's structure. Could you run a separate bash script that sources nvm.sh, and output the version path, and run that from the Makefile?

Comment (danbev) [reply locale/en/download/package-manager.md]: > Could you run a separate bash script that sources nvm.sh, and output the version path, and run that from the Makefile?

Would be enough just to pass a dummy version to `nvm_version_path` and use the path, without the dummy version, and then send that path to make? 

Something along the lines of : 
```shell
$ make install DESTDIR=`nvm_version_path 999|sed s/999$//` PREFIX='$(FULLVERSION)-pre'
```

Comment (ljharb) [reply locale/en/download/package-manager.md]: No, because `nvm_version_path` does different things depending on the version number.

Comment (danbev) [reply locale/en/download/package-manager.md]: > No, because nvm_version_path does different things depending on the version number.

After running `declare -f nvm_version_path` I see what you mean. Thanks

So another attempt:
```shell
$ env VERSION=`python tools/getnodeversion.py` make install DESTDIR=`nvm_version_path v$VERSION` PREFIX=""
```
Perhaps not pretty but might an acceptable solution until support for this lands in nvm like you mentioned before?

Comment (ljharb) [reply locale/en/download/package-manager.md]: That seems much better, thanks!

**Review by ljharb — APPROVED** [review]
Thanks!

**Comment by danbev** [comment]
Would someone will commit rights to this repo be willing to merge and push this or grant me commit rights so I can push it?

**Comment by sotayamashita** [comment]
@fhemberger Could you review and merge it, if no problem.
cc: @nodejs/website

File: locale/en/download/package-manager.md
Comment (fhemberger) [inline locale/en/download/package-manager.md]: Could you please add a sentence or two about installing nvm in general and using it to install Node.js, not just how to add builds from source to it? Thx.

Otherwise LGTM.

**Review by fhemberger — CHANGES_REQUESTED** [review]

File: locale/en/download/package-manager.md
Comment (ljharb) [reply locale/en/download/package-manager.md]: Please strongly stress using the sole supported installation method - the curl command in the readme - over alternative unsupported methods of installation :-)

Comment (danbev) [reply locale/en/download/package-manager.md]: @fhemberger @ljharb I've added some information in general (very short) about nvm and install instructions. Let me know what you think. Thanks

**Review by fhemberger — APPROVED** [review]

**Comment by fhemberger** [comment]
LGTM!

**Comment by fhemberger** [comment]
Thanks again!

**Pull request merged**
